### DISCUSSION TITLE
Option to have more a "server" like space

### DISCUSSION THREAD

**Discussion by Sporesirius** [element-hq/element-meta#660] [Ideas]
#### What would you like to do?
- A space option where you automatically in all rooms.
- A new room option where we can see the users under the "Rooms" section that are in a call (a bit like discord or teamspeak 3), so you don't "make a call" but just join the room.
- A user list of all members of the space (perhaps also sorted by the space permission group).

#### Why would you like to do it?
- Currently, in a space where you want voice or video, you have to manually join each room (joining also adds a bit delay).
- When you make a call in a room, the call is displayed to each member that has joined the room.
- It is a bit difficult to see who and how many people are in the room

**Comment by LuisAPI** [comment]
A simple "Join all rooms" button in a space would be easy to implement but make a lot of difference
